examples/simple_repeater/main.cpp

```diff
@@ -247,6 +308,69 @@ class MyMesh : public mesh::Mesh, public CommonCLICallbacks {
       radio_set_params(_prefs.freq, _prefs.bw, _prefs.sf, _prefs.cr, _prefs.sync_word);
       MESH_DEBUG_PRINTLN("Radio params restored");
     }
+
+    #ifdef ENABLE_BLE
+
+    if(_prefs.ble_enabled && !bleScan->isScanning() && !bleReported){
+
+      bleReported = true;
+      NimBLEScanResults results = bleScan->getResults();
+
+
+      for(int i=0; i<results.getCount(); i++){
+        const NimBLEAdvertisedDevice* advertisedDevice = results.getDevice(i);
+
+        float rssi = (float) advertisedDevice->getRSSI();
+
+        uint8_t raw[300];
+        uint16_t rawLength = 0;
+
+        const uint8_t* addrBuf = advertisedDevice->getAddress().getVal();
+        raw[0] = addrBuf[5];
+        raw[1] = addrBuf[4];
+        raw[2] = addrBuf[3];
+        raw[3] = addrBuf[2];
+        raw[4] = addrBuf[1];
+        raw[5] = addrBuf[0];
+        rawLength = 6;
+
+        const uint8_t* payloadBuf = advertisedDevice->getPayload().data();
+        uint16_t payloadLength = advertisedDevice->getPayload().size();
+        memcpy(raw+rawLength, payloadBuf, payloadLength);
+        rawLength += payloadLength;
+
+        CLIMode cli_mode = _cli.getCLIMode();
+        if (cli_mode == CLIMode::CLI) {
+
+          CommonCLI* cli = getCLI();
+          Serial.printf("%lu", rtc_clock.getCurrentTime());
+          Serial.printf(",RXBLE,%.2f,%.2f,", rssi, 0.0f);
+          mesh::Utils::printHex(
+            Serial,
+            raw,
+            rawLength
+          );
+          Serial.println();
+
+        } else if (cli_mode == CLIMode::KISS) {
+
+          uint8_t kiss_rx[CMD_BUF_LEN_MAX];
+          KISSModem* kiss = this->getCLI()->getKISSModem();
```

Comment:
remove `this->`, `getCLI()` is in scope here, I think you copied some of my code here, so if you find the one you copied, please fix it